src/julia_threads.h

```diff
@@ -124,6 +124,14 @@ typedef struct {
 
 struct _jl_bt_element_t;
 
+typedef struct {
+    uint64_t start_time;
+    uint64_t sleep_time;
+    uint64_t scheduler_time;
+    uint64_t lock_spin_time;
+    uint64_t gc_time;
+} jl_timing_tls_states_t;
```

Comment:
TODO: should this be like GC_Num and have a corresponding struct on the Julia side, so on that side we work with the struct rather than individual numbers?

Reply:
downside: any user-facing struct is impossible to expand, so probably just want to be exposing functions that return numbers... could still be passing data from C -> Julia side as a struct but idk if that gains us much tbh